src/Domain/Entity/Group.php

```diff
@@ -4,5 +4,13 @@
 
 abstract class Group extends CoreEntity
 {
+    public function isTlec(): bool
+    {
+        return true;
+    }
 
+    public function isTleo(): bool
+    {
+        return true;
```

Comment:
We've now got a setup where Groups may have parents (e.g. a Gallery may belong to an Episode) so they should follow the same logic as everything else - check the parent, so this can be removed